### DISCUSSION TITLE
Is it possible to load assets from an archive?

### DISCUSSION THREAD

**Discussion by Barugon** [FyroxEngine/Fyrox#268] [Q&A] [answered]
If not, would this be something that could be added (perhaps [mla](https://github.com/ANSSI-FR/MLA))?

**Comment by mrDIMAS** [comment, marked as answer]
No, it is not possible yet, but indeed it would good to have such feature.

**Reply by DuckEater54** [reply]
Another thought: Using DLLs to store assets. I don't really understand how it works but I know DLLs can store assets because assigning custom icons to shoulders on windows uses a DLL file path
